sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -138,13 +145,13 @@ export interface FieldData {
 export type FormContentType = "application/pdf" | "image/jpeg" | "image/png" | "image/tiff";
 
 // @public
-export type FormElement = FormWord | FormLine;
+export type FormElement = FormWord | FormLine | FormSelectionMark;
 
 // @public
 export interface FormElementCommon {
     boundingBox: Point2D[];
     pageNumber: number;
-    text: string;
+    text?: string;
```

Comment:
should we remove `text` all together from this?

Reply:
Well, the reason for leaving it here is that it ensures that `text` remains a property on all `FormElement` variants going forward. I originally thought to just make `text` a required property of both `FormWord` and `FormLine`, but that would result in anyone who has accessed the `text` property of a `FormElement` to get a new error about the property not _existing_, which seems worse than if someone were to get an error about it possibly being undefined in some cases. Essentially, in the previous contract there was a guarantee that `text` would be a property that exists and we decided that since we can't keep the type compatible all the way, we decided that we need to maintain the existence of the property.

Reply:
ah I see, so it is to maintain backward compatibility.